.github/workflows/build.yml

```diff
@@ -1,90 +1,106 @@
 name: Github CI
 
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+    push:
```

Comment:
Not a fan of the formatting in this file. I think you've got it set to 4 spaces per tab. I would prefer 2 spaces like before.

Reply:
@ejgallego said that he prefers 4 spaces, so we should be uniform in our formatting: we can't have 2 spaces for .yml files, and 4 spaces for .ts files.

Reply:
Actually, we format .ml files with 2 spaces, so I'm not sure why @ejgallego said that he prefers 4 spaces. I'll update prettier to use 2 spaces, and see if @ejgallego agrees with it.

Reply:
For ts files that is fine, but for yml, we quickly run out of room to read files on small screens.